vendor/github.com/ovirt/terraform-provider-ovirt/ovirt/resource_ovirt_image_transfer.go

```diff
@@ -38,7 +38,7 @@ func resourceOvirtImageTransfer() *schema.Resource {
 		},
 
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(10 * time.Minute),
+			Create: schema.DefaultTimeout(20 * time.Minute),
```

Comment:
can you use timeout section in the terraform files to override this value?

Reply:
I wasn't aware that is possible, but now that you pointed it I see that the timeout block gives us that ability.
I'll fix and revert the patch to the terraform provider.
Thanks for the catch